WPMedia/arc-themes-components
# Fixed minimize screen issue

## Ticket

- [THEMES-1023](https://arcpublishing.atlassian.net/browse/THEMES-1023)

## Description

Article Body block - by clicking, expand then Esc, the expand button should display as an X icon with expand text. 

https://dev.themesinternal.arcpublishing.com/pf/article-test/?_website=the-gazette

## Acceptance Criteria

Article Body block - by clicking, expand then Esc, the expand icon button should display.

## Test Steps

1. Checkout branch - `git checkout THEMES-1023`
3. Run Storybook `npm run storybook`
4. Check out the carousel storybook expand and esc 

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1023]: https://arcpublishing.atlassian.net/browse/THEMES-1023?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
